### ISSUE TITLE
Investigate indentation for function call continuations

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1241]
See http://sourceforge.net/p/uncrustify/bugs/651/

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\10062-UNI-1356.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\10062-UNI-1356.cpp"
index 99247e3..0be72ae 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\10062-UNI-1356.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\10062-UNI-1356.cpp"
@@ -1,27 +1,27 @@
 // Hi,
 // When using "space only" and "indent continue", I notice a wrong indentation in C language (at least)
 // function call when the retrun value is assigned to a variable and the call is split in two or more line.
 // In that case the indent is twice the indent set in "indent_continue"
 // This only appears when "indent_with_tabs" is set to 0 "space only" and 1
 // "indent with tabs to brace level, align with spaces"
 // Version tested:
 // 0.59: good indentation
 // 0.60: wrong indentation
 // master (sha1 fc5228e): wrong indentation
 // Here are some details about thats issue:
 // orignal code
 // The long line are manually split and not indented to test uncrustify indent
 
 int main (int argc, char *argv[])
 {
   double a_very_long_variable = test (foobar1, foobar2, foobar3, foobar4,
-      foobar5, foobar6);
+          foobar5, foobar6);
 
   double a_other_very_long = asdfasdfasdfasdfasdf + asdfasfafasdfa +
       asdfasdfasdf - asdfasdf + 56598;
 
   testadsfa (dfasdf, fdssaf, dsfasdf, sadfa, sadfas, fsadfa,
       aaafsdfa, afsd, asfdas, asdfa, asfasdfa, afsda, asfdasfds, asdfasf);
 
   return 0;
 }
```

Debug support files:
---
[10062-UNI-1356.cpp](https://gist.githubusercontent.com/mihaipopescu/5197636a3ea4f4c8c348e0ba81d1e42e/raw/99247e39688d238c19cee099699e50781cdeac44/10062-UNI-1356.cpp)
[10062-UNI-1356.cpp.log](https://gist.githubusercontent.com/mihaipopescu/5197636a3ea4f4c8c348e0ba81d1e42e/raw/9c96113305332e1607823f051706dedc71a90a20/10062-UNI-1356.cpp.log)
[10062-UNI-1356.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/5197636a3ea4f4c8c348e0ba81d1e42e/raw/be5f3816c2c92bbb8912d694ad8b7754d92b74d0/10062-UNI-1356.cpp.unc)
[UNI-1356.cfg](https://gist.githubusercontent.com/mihaipopescu/5197636a3ea4f4c8c348e0ba81d1e42e/raw/3d3a740bcf08203328e909c2d5bf983076a57cc9/UNI-1356.cfg)
[UNI-1356.cpp](https://gist.githubusercontent.com/mihaipopescu/5197636a3ea4f4c8c348e0ba81d1e42e/raw/0225ca96396074e3087078342abc176ef676ea4e/UNI-1356.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by gmaurel** [comment]
This is a uncrustify feature.

Now you can choose the default use of it (TWO indentations)
or have
only ONE indentation (as you wish) by using a new option:
use_indent_continue_only_once = true

**Comment by mihaipopescu** [comment]
@gmaurel I would also recommend before closing the issues we could also add the test into `imported.test` testing suite. What do you think ?

**Comment by gmaurel** [comment]
I can read in `tests/CMakeLists.txt` at line 21:
`foreach(test_lang c-sharp c cpp d java pawn objective-c vala ecma imported)`
Is it what you wanted to have?

**Comment by mihaipopescu** [comment]
Yes, I meant adding this tests to imported test suite.

**Comment by guy-maurel** [comment]
Using ctest one can see:
```
        Start 752: imported_10001
752/761 Test #752: imported_10001 ...................   Passed    0.01 sec
        Start 753: imported_10002
753/761 Test #753: imported_10002 ...................   Passed    0.02 sec
```
and many more.
Only the file `tests/run_tests.py` doesn't contain imported.
I'll make a PR for that.

**Comment by guy-maurel** [comment]
see PR #1327

**Comment by mihaipopescu** [comment]
Thanks ... but I wanted to say that we need to add the files from every bug we fix/close as a new test.

**Comment by guy-maurel** [comment]
May I close?

**Comment by gmaurel** [comment]
Fixed at b74aeafaff7eea2b0010b87c769e222b25777fea, Dec  8 18:49:59 2017
